### PR TITLE
Indicate private modules in output from `updated`

### PR DESCRIPTION
**Pull request by gigabo**
This is helpful for figuring out what will actually be published.

Goes along with #321.

Looks like:

image

### CODE REVIEW

**Comment by hzoo** [comment]
👍

File: src/commands/UpdatedCommand.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ export default class UpdatedCommand extends Command {
 
   execute(callback) {
     const formattedUpdates = this.updates
-      .map((update) => `- ${update.package.name}`)
+      .map((update) => `- ${update.package.name}${update.package.isPrivate()?" (private)":""}`)
```
Comment (hzoo) [inline src/commands/UpdatedCommand.js]: maybe spaces between the operators though

Comment (gigabo) [inline src/commands/UpdatedCommand.js]: Oh, yeah.  Weird that the linter didn't catch that...

Comment (gigabo) [inline src/commands/UpdatedCommand.js]: Looks like that linter rule got dropped accidentally.  [Re-added](https://github.com/lerna/lerna/commit/309029397fc6f4a7bac7020af2ba4313c4bf1e4b) and fixed this up.

Nice catch @hzoo!

**Pull request merged**